docs/en/oas3/OAS3-PDND-AS.yaml

```
@@ -0,0 +1,313 @@
+                  - type: object
+                    properties:
+                      error:
+                        type: string
+                        description: invalid_dpop_proof
+                      error_description:
+                        type: string
+                        description: >-
+                          The request cannot be fulfilled because it contains an
+                          invalid dpop proof.
+                    required:
+                      - error
+              examples:
+                invalid request:
+                  value:
+                    error: invalid_request
+                    error_description: >-
+                      The request cannot be fulfilled because it is missing
+                      required parameters, contains invalid parameters, or is
+                      otherwise malformed
+                invalid dpop proof:
+                  value:
+                    error: invalid_dpop_proof
+                    error_description: >-
+                      The request cannot be fulfilled because it contains an
+                      invalid dpop proof
+        '401':
+          description: Unauthorized
+          headers:
+            WWW-Authenticate:
+              description: The request cannot be fulfilled because the Voucher is expired, revoked or otherwise malformed. See <a target="blank" href="https://datatracker.ietf.org/doc/html/rfc6750.html#section-3">RFC6750</a> and <a target="blank" href="https://datatracker.ietf.org/doc/html/rfc9449.html#section-7.1-11">RFC9449</a> for details.
+              schema:
+                type: string
+        '404':
+          description: Claims not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    description: The error code
+                    example: not_found
+                  error_description:
+                    type: string
+                    description: >-
+                      Text in human-readable form providing further details to
+                      clarify the nature of the error encountered
+                    example: >-
+                      The authentic source cannot fulfill the request because the
+                      claims were not found
+                required:
+                  - error
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    description: server_error
+                  error_description:
+                    type: string
+                    description: >-
+                      The request cannot be fulfilled because the e-Service Endpoint encountered an internal problem.
```

- Set mandatory `error` value as an `enum` instead of `example`;
- Specify meaningful descriptions for `error` and `error_description` (instead of specific values).

```suggestion
                  error:
                    type: string
                    description: The error code
                    enum: [server_error]
                  error_description:
                    type: string
                    description: >-
                      Text in human-readable form providing further details to
                      clarify the nature of the error encountered
```